src/documents/tools/Prototyping.md

```diff
@@ -23,12 +23,12 @@ testing.
 
 ## Try
 
-* There are several digital tools that help designers create interactive prototypes, many of which are available online.
+1. There are several digital tools that help designers create interactive prototypes, many of which are available online.
   For example, InVision is a prototyping tool that is easy to learn and use.
-* If you are designing a product, you can use 3D printing to build high fidelity prototypes.
-* If you are working on a digital product, such as a website or an application, try publishing a beta version to get
+2. If you are designing a product, you can use 3D printing to build high fidelity prototypes.
+3. If you are working on a digital product, such as a website or an application, try publishing a beta version to get
   user feedback and fix the technical issues.
-* If you are designing a service or a system, try roleplaying sessions to test your proposed solution.
+4. If you are designing a service or a system, try roleplaying sessions to test your proposed solution.
```

Comment:
I'm not entirely sure this should be an ordered list. It doesn't appear to be ordered steps for example, but I'm not sure if you are meant to try one before the other. @dayotte and @jhung can you advise. It's probably worth while to review all of the "Try" sections to indicate how to best present the lists. That might be work for a new issue if that hasn't been modified in this PR.

Reply:
@jobara I looked at the other "Try" sections and they all appear to use ordered lists. But yes, this should probably be a separate issue as it doesn't have anything to do with the line lengths per se.

Reply:
@greatislander @jobara looking through the guide again I can see the confusion. I'm thinking that only the step-by-step "Try" sections should be numbered. These are mostly in the Tools section (but not all Tools entries). Others should be bulleted (e.g. most of the Practices entries). In the Insights section they are just paragraphs. I can specify  which are which if that would help!

Reply:
@dayotte thanks, it's probably worth filing a GitHub issue with the details about what should be what and how they should look.